src/plugins/data_table/server_side_data_table.js

```diff
@@ -70,13 +70,49 @@ var ServerSideDataTable = (function() {
 
     // dataTables callback to allow addition of params to the ajax request
     _addServerParams : function(aoData) {
+      if (this.simpleParams) {
+        var sortBy, sortDir, limit, start, requestId;
+
+        var indexOfSortedColumn = this._getDataTableParam(aoData, "iSortCol_0");
```

Comment:
good question, yes we do lose that flexibility. I'll look into a nice way to extend this to support arbitrary sort columns